avocado_vt/loader.py

```diff
@@ -16,6 +16,7 @@
 Avocado VT plugin
 """
 
```

Comment:
It's not because it's not being called a refactor, that it's not a refactor.  The introduction of `add_basic_vt_options()` in place of the corresponding lines of code and making it shared among vt and vt-list is a refactor.  That's even the title of the commit.

"While on it", signals a change in objective, and "adds support for --vt-arch to the vt-list plugin as well as other basic vt options" is a clear secondary change.

You may prove me wrong with a split commit that looks hideous, but at this point, I'm pretty sure we're talking about two changes here.